corona-warn-app/cwa-app-android
# Contact Journal Extension - Store RAT and PCR Tests results (EXPOSUREAPP-6041)

- Test result is added to a new database and later will be displayed in the contact journal 
 
### Testing
- Scan PCR/RAT test
- See test results
- If the test is positive or negative new record should be added to DB
- Check Test Menu > Corona Tests > Contact diary tests if the test is there
